packages/tools/devtools/devtools/src/index.ts

```diff
@@ -144,7 +144,9 @@ class Devtools implements IDevtools {
 }
 
 /**
- * {@inheritDoc @fluid-experimental/devtools-core#initializeDevtoolsBase}
```

Comment:
Unfortunately, API-Extractor can't validate that the item being inherited from has documentation because the symbol lives in another package, so this triggers a false positive. See: https://github.com/microsoft/rushstack/issues/3462

Reply:
In this case, it probably makes sense for this item to have its own docs anyways. The semantics are the same for now, but I suspect that will change.